src/org/opensolaris/opengrok/analysis/csharp/CSharpXref.lex

```diff
@@ -43,25 +43,17 @@ import org.opensolaris.opengrok.web.Util;
 %int
 %include CommonXref.lexh
 %{
-  /* Must match WhiteSpace regex */
-  private final static String WHITE_SPACE = "[ \t\f\r]+";
-
   // TODO move this into an include file when bug #16053 is fixed
   @Override
   protected int getLineNumber() { return yyline; }
   @Override
   protected void setLineNumber(int x) { yyline = x; }
 %}
 
-/* Must match WHITE_SPACE constant */
-WhiteSpace     = [ \t\f\r]+
-EOL = \r|\n|\r\n|\u2028|\u2029|\u000B|\u000C|\u0085
+CsharpEOL = {EOL}|\u2028|\u2029|\u000B|\u000C|\u0085
```

Comment:
we should finally fully support UTF, so I hope we will end up with common UTF eol detection regexp eventually